### ISSUE TITLE
Debug broken in VScode version 1.47.1

### ISSUE THREAD

**Issue by Moser-ss** [jest-community/vscode-jest#605]
### Environment
MacOS 10.15.5
VSCode 1.47.1
### Prerequisite
debug config
```
        {
            "type": "node",
            "name": "vscode-jest-tests",
            "request": "launch",
            "program": "${workspaceFolder}/node_modules/jest/bin/jest",
            "args": [
              "--runInBand",
              "--config",
              "test/jest.config.json"
            ],
            "cwd": "${workspaceFolder}",
            "console": "integratedTerminal",
            "internalConsoleOptions": "neverOpen",
            "disableOptimisticBPs": true
          },
```
### Steps to Reproduce
Add a breakpoint in the file of the code we are testing
Debug the test using the vscode-jest-tests config

### Expected Behavior
The debugger stops in the breakpoint set in the code's file

### Actual Behavior
Debugger skips the breakpoint set in the code's file
--- 
This issue started when I upgrade from 1.46.1 to 1.47.1. if I downgrade the VSCode to 1.46.1 it will work again

The fastest (and the most fun) way to resolve the issue is to submit a pull-request yourself. If you are interested, feel free to check out the [contribution guide](https://github.com/jest-community/vscode-jest/CONTRIBUTING.md), we look forward to seeing your PR...

**Comment by mjashanks** [comment]
Can confirm this exact issue... downgraded and they worked again.

However, this seems to be the case with all the other jest extensions - none of them work..

**Comment by csalvato** [comment]
@Moser-ss Thank you so much for this post. I was going crazy trying to figure out why my debugger wasn't working. Downgrading to 1.46.1 resolved the issue for me, as well.

Here's to hoping this gets resolved 🙏

**Comment by connectdotz** [comment]
does it still happen for vscode 1.48.x? I can't reproduce it on my mac with 1.48.2. Do you guys have a sample repo to demonstrate this behavior?

**Comment by csalvato** [comment]
@connectdotz I was on vscode 1.48.2 before downgrading and was experiencing the same problem.

Do you not get this issue on a fresh CRA app?

**Comment by connectdotz** [comment]
> Do you not get this issue on a fresh CRA app?

no, the breakpoint worked as expected in a freshly created CRA project...

**Comment by csalvato** [comment]
I am now on 1.49.0 and the issue seems to be resolved 🤔 

1.48.2 was causing issues for me, though. 🤷 

Seems like it's resolved in 1.49, though, so maybe this can be closed.

Thanks @connectdotz
